lib/triggerParser.js

```diff
@@ -3,27 +3,30 @@
 'use strict';
 
 var extractTriggers = require('./extractTriggers');
+var EXIT = function() { process.exit(0); };
 
-var packageDir = process.argv[2];
-if (!packageDir) {
-  process.send({error: 'Must supply package directory for functions trigger parsing.'});
-  process.exit(0);
-}
+(function() { // wrap in function for flow control
```

Comment:
It wasn't immediately clear to me what "for flow control" meant here. I suggest clarifying to "to allow exiting this code (returning from the function) without exiting the process".